electron/locale/locale.js

```diff
@@ -44,27 +43,27 @@ const tr = require('./strings-tr');
 const uk = require('./strings-uk');
 
 const SUPPORTED_LANGUAGES = {
-  'en': 'English',
-  'cs': 'Čeština',
-  'da': 'Dansk',
-  'de': 'Deutsch',
-  'el': 'Ελληνικά',
-  'es': 'Español',
-  'fr': 'Français',
-  'hr': 'Hrvatski',
-  'it': 'Italiano',
-  'lt': 'Lietuvos',
-  'hu': 'Magyar',
-  'nl': 'Nederlands',
-  'pl': 'Polski',
-  'pt': 'Português do Brasil',
-  'ro': 'Română',
-  'ru': 'Русский',
-  'sk': 'Slovenčina',
-  'sl': 'Slovenščina',
-  'fi': 'Suomi',
-  'tr': 'Türkçe',
-  'uk': 'Українська',
+  cs: 'Čeština',
+  da: 'Dansk',
+  de: 'Deutsch',
+  el: 'Ελληνικά',
+  en: 'English',
+  es: 'Español',
+  fi: 'Suomi',
+  fr: 'Français',
+  hr: 'Hrvatski',
+  hu: 'Magyar',
+  it: 'Italiano',
+  lt: 'Lietuvos',
+  nl: 'Nederlands',
+  pl: 'Polski',
+  pt: 'Português do Brasil',
+  ro: 'Română',
+  ru: 'Русский',
+  sk: 'Slovenčina',
+  sl: 'Slovenščina',
+  tr: 'Türkçe',
+  uk: 'Українська',
```

Comment:
These were sorted by value with English on top on purpose as it defines the language order in menu.